pkg/controllers/resource_map.go

```diff
@@ -28,4 +29,7 @@ var (
 
 	// a map to track active stores
 	StoreMap = rs.NewActiveStores()
+
+	// a map between CertificateStore name to array of x509 certificates
+	CertificatesMap = cs.NewActiveCertStores()
```

Comment:
should we rename this variable since the key of the map is stores?

Reply:
good call, I'll rename it